src/builder/create_embed.rs

```diff
@@ -269,6 +269,12 @@ impl From<EmbedAuthor> for CreateEmbedAuthor {
     }
 }
 
+impl From<User> for CreateEmbedAuthor {
+    fn from(user: User) -> Self {
+        Self::new(&user.name).icon_url(user.face())
```

Comment:
You should pass user.name is as-is, to avoid cloning.
```suggestion
        Self::new(user.name).icon_url(user.face())
```

Reply:
Call user.face before passing in the name then, I should have spotted that when making the suggestion lol.

Reply:
Yeah also bad call from me to just commit that. But CreateEmbedAuthor::new uses user.name as argument so I guess I should just make it without the builder function via From<EmbedAuthor> but wouldn't that also clone?